examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Express examples
+
+This page contains list of examples using Express.
+
+- Authentication with login and passowrd - [examples/auth](./auth)
+- HTTP content negotiation - [examples/content-negotiation](./content-negotiation)
+- Working with cookie-based session - [examples/cookie-sessions](./cookie-sessions)
+- Working with cookies via [`cookie-parser`](https://github.com/expressjs/cookie-parser) - [examples/cookies](./cookies)
```

Comment:
This is just a generic cookie example, nothing specific with cookie-parser. It is to demo the res.cookie and res.clearCookie APIs in Express

Reply:
I've changed it to just `Working with cookies`